webapp/tests/karma/unit/controllers/reports.js

```diff
@@ -142,12 +142,13 @@ describe('ReportsCtrl controller', () => {
       scope.$broadcast('VerifyReport', true);
       return Promise.resolve().then(() => {
         chai.expect(post.callCount).to.equal(1);
-        chai.expect(post.args[0]).to.deep.equal([{
+        chai.expect(post.args[0][0]).to.deep.include({
           _id: 'def',
           name: 'hello',
           form: 'P',
           verified: true,
-        }]);
+        });
+        chai.expect(parseInt(post.args[0][0].verified_date)).to.be.closeTo(Date.now(), 1000);
```

Comment:
By using `sinon.useFakeTimers` you can make this test precise.

Reply:
`useFakeTimers` is causing a few failures in other tests which rely on setTimeout. Instead of handling that, I'm just mocking `Date.now()` -- hope that suites.